README.md

```diff
@@ -592,43 +588,107 @@ You can find more detailed API documentation in Doxygen reference: https://eosio
 
-Now you can start the Docker container:
+*Note:* If creating an account, you should generate a key for each permission of the account. 
+
+**Important:**
+Save these values in a safe place for future reference.
+
+<a name="generaluse-createaccount"></a>
+#### Creating an account
+
+Creating an account requires that you have keypairs to set as `owner` and `active` permissions. 
+
+Run `create` command where `owner_public_key` and `active_public_key` are the *public keys* generated by the `create key` command:
+
+_Note: Keys are generated arbitrarily, it's up to you which key is used for a particular account permission._
 
 ```bash
-sudo rm -rf /data/store/eos # options 
-sudo mkdir -p /data/store/eos
-docker-compose -f Docker/docker-compose.yml up
+./eosc create account inita bob owner_public_key active_public_key 
 ```
 
-Get chain info:
+You should get a json response back with a transaction ID confirming it was executed successfully.
+
+<a name="generaluse-accountexists"></a>
+#### Checking Existence of Account
+
+Check that account an account exists or was created successfully. 
 
 ```bash
-curl http://127.0.0.1:8888/v1/chain/get_info
+./eosc get account bob
 ```
 
-<a name="dockercontract"></a>
-### Running contract in docker example 
+You should get a response similar to this:
 
-You can run the `eosc` commands via `docker exec` command. For example:
+```json
+{
+  "name": "bob",
+  "eos_balance": 0,
+  "staked_balance": 1,
+  "unstaking_balance": 0,
+  "last_unstaking_time": "2106-02-07T06:28:15"
+}
+```
+
+<a name="generaluse-walletimport"></a>
+#### Importing
+
+`wallet import` accepts a private key as the first positional argument, the private key imported should be for the **active** permission of the account you wish to import.
 
 ```bash
-docker exec docker_eos_1 eosc set contract exchange build/contracts/exchange/exchange.wast build/contracts/exchange/exchange.abi
+./eosc wallet import 5XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 ```
+
+The **bob** account is now imported into your wallet
+
+
+<a name="generaluse-pushmessage"></a>
+#### Push message to Contract
+
+To send a message to a contract, we need to specify the **contract** and its **method** we are pushing the message to, the **parameters** anticipated by the contract's method and then some additional arguments depending on the nature of the message. 
+
+The below example pushes a message to the **transfer** method of the sample **currency** contract from user **bob**, it will include parameters that specify the from (*bob*) and to (*inita*) account, and the amount to transfer (*50*), we'll set the `--scope` of the message to include these two accounts `bob,inita`, and the `--permissions` required to send the message, which are bob's active permissions, expressed with `bob@active`.
+
+```./eosc push message grocery transfer '{"from":"grocery","to":"bob","amount":50}' --scope grocery,inita --permission grocery@activex```
```

Comment:
activex -> active

Reply:
also, shouldn't it be push message currency transfer?

Reply:
and is the scope right?

Reply:
That's an artifact from another branch, thought that wasn't included.